### PR TITLE
Fix: Assimp generates normals for all the meshes.

### PR DESCRIPTION
**Pull request by onurtore**
Some mesh files does not have normals and this creates exceptions when we try to load them into memory. Using the generate normal flag fixes this issue.

### CODE REVIEW

**Comment by onurtore** [comment]
@luca-della-vedova

File: graphics/src/MeshManager.cc [hunk 1]
```diff
@@ -146,19 +146,23 @@ const Mesh *MeshManager::Load(const std::string &_filename)
...
     else if (extension == "dae")
       loader = &this->dataPtr->colladaLoader;
     else if (extension == "obj")
-      loader = &this->dataPtr->objLoader;
+      //loader = &this->dataPtr->objLoader;
+      loader = &this->dataPtr->assimpLoader;
+    else if (extension == "gltf")
+      loader = &this->dataPtr->assimpLoader;
+    else if (extension == "fbx")
+      loader = &this->dataPtr->assimpLoader;
```
Comment (luca-della-vedova) [inline graphics/src/MeshManager.cc:160]: Any specific reason for this change? My hope (if possible!) was to be able to substitute all the loaders with the assimp loader and I already got most static DAE assets, as well as some animated ones, to work fine. We can fallback later to the other loaders if we find an issue that can't be handled by assimp if that makes sense?

File: graphics/src/AssimpLoader.cc [hunk 2]
```diff
@@ -307,6 +306,7 @@ Mesh *AssimpLoader::Load(const std::string &_filename)
       aiProcess_RemoveRedundantMaterials |
       aiProcess_SortByPType |
       aiProcess_Triangulate |
+      aiProcess_GenNormals |
```
Comment (luca-della-vedova) [inline graphics/src/AssimpLoader.cc:309]: Do you have an example asset where this happens?

Comment (onurtore) [reply graphics/src/AssimpLoader.cc:309]: Adding it below: 
[airbot.zip](https://github.com/gazebosim/gz-common/files/8751580/airbot.zip)

File: graphics/src/MeshManager.cc (hunk 1 above)
Comment (onurtore) [reply graphics/src/MeshManager.cc:160]: Yes sure, should I git force push to remove the commit, or does GitHub gives you option to select commits to merge?

Comment (luca-della-vedova) [reply graphics/src/MeshManager.cc:160]: I can't really do that it's either merge all or nothing!

Comment (onurtore) [reply graphics/src/MeshManager.cc:160]: Okay, gonna fix it later today.

Comment (onurtore) [reply graphics/src/MeshManager.cc]: @luca-della-vedova  fixed.

**Comment by onurtore** [comment]
@chapulina,
Can I add labels? Its looks little bit exhausting for you to add label for each PR.

**Pull request merged**